tests/onnx/quantization/test_classification_models_graph.py

```diff
@@ -17,29 +17,31 @@
 from torchvision import models
 import onnx
 # pylint: disable=no-member
-
+from tests.onnx.conftest import ONNX_MODEL_DIR
 from tests.onnx.quantization.common import ModelToTest
 from tests.onnx.quantization.common import min_max_quantize_model
 from tests.onnx.quantization.common import compare_nncf_graph
 from tests.onnx.quantization.common import mock_collect_statistics
+from tests.onnx.weightless_model import load_model_topology_with_zeros_weights
+
+TORCHVISION_TEST_DATA = [(ModelToTest('resnet18', [1, 3, 224, 224]), models.resnet18(pretrained=True)),
```

Comment:
Why did you change the test and the reference?

Reply:
I add additional test for densenet model from onnx model zoo, which has the pattern that I added. Because of that I divided the test onto two: models from torchvision and onnx models